
Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET / HTTP/1.1" 500 145
Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET /favicon.ico HTTP/1.1" 500 145
Internal Server Error: /
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET / HTTP/1.1" 500 145
Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET /favicon.ico HTTP/1.1" 500 145
Internal Server Error: /
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET / HTTP/1.1" 500 145
Internal Server Error: /favicon.ico
Traceback (most recent call last):
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\middleware.py", line 12, in process_request
    request._messages = default_storage(request)
                        ^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\__init__.py", line 12, in default_storage
    return import_string(settings.MESSAGE_STORAGE)(request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 16, in __init__
    self.storages = [
                    ^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\fallback.py", line 17, in <listcomp>
    storage_class(*args, **kwargs) for storage_class in self.storage_classes
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\contrib\messages\storage\cookie.py", line 77, in __init__
    self.signer = signing.get_cookie_signer(salt=self.key_salt)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\core\signing.py", line 114, in get_cookie_signer
    key=_cookie_signer_key(settings.SECRET_KEY),
                           ^^^^^^^^^^^^^^^^^^^
  File "I:\GeekBrains\GB_Django\GB_django_seminar6\venv\Lib\site-packages\django\conf\__init__.py", line 111, in __getattr__
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
"GET /favicon.ico HTTP/1.1" 500 145
I:\GeekBrains\GB_Django\GB_django_seminar6\gbhw\mysite\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
I:\GeekBrains\GB_Django\GB_django_seminar6\gbhw\mysite\settings.py changed, reloading.
Watching for file changes with StatReloader
I:\GeekBrains\GB_Django\GB_django_seminar6\gbhw\mysite\settings.py changed, reloading.
